### DISCUSSION TITLE
[Support]: Move 'Select From Timeline' to Top

### DISCUSSION THREAD

**Discussion by dopeytree** [blakeblackshear/frigate#13113] [General Support]
### Describe the problem you are having

Its a bit odd to have 1hour as the default export time.

I suggest we move the option 'Select From Timeline' to the TOP.

Generally users will be exported clips of a minutes or less i.e associated with the detection length.

### Steps to reproduce

export video

### Version

0.14

### Frigate config file

```yaml
n/a
```

### Relevant log output

```shell
n/a
```

### Operating system

UNRAID

### Install method

Docker Compose

### Network connection

Wired

### Camera make and model

annie c500

**Comment by NickM-27** [comment]
This is not a bug

**Reply by dopeytree** [reply]
Ok but what is it? an idea or discussion?

**Reply by NickM-27** [reply]
Idea would be more appropriate
